### ISSUE TITLE
apt ppa adding failed

### ISSUE THREAD

**Issue by kenysc99** [fetzerch/hp-n54l-drivers#4]
Hi Christian,

Thank you very much for this great work, I love my n54l and you're giving she a new life!

I'm trying to install your driver using the ppa but was getting the 404 error as below, is there any way round?

```
Get:7 http://ppa.launchpad.net/team-xbmc/ppa/ubuntu bionic InRelease [15.4 kB] 
Err:8 http://ppa.launchpad.net/fetzer-ch/hp-n54l/ubuntu bionic Release         
  404  Not Found [IP: 91.189.95.83 80]

E: The repository 'http://ppa.launchpad.net/fetzer-ch/hp-n54l/ubuntu bionic Release' does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.

```
Cheers,

Ken

**Comment by fetzerch** [comment]
Hi Ken,

the patches have been merged into Kernel 4.5, that's why for Bionic there's no need to install the package anymore. You just need to configure your system according to the config files: https://github.com/fetzerch/hp-n54l-drivers/tree/master/config.

There's unfortunately a conflict that you can't have both sensors and watchdog working at the same time. Therefore watchdog needs to be blacklisted. (see config)

Cheers,
Christian

**Comment by kenysc99** [comment]
Hi Christian, followed the "config" and I have the sensors working all fine, thank you very much!
